student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown')
+    super(age, name)
+    @classroom = classroom
```

Comment:
Please add `parent_permission = true` as the parameter in the constructor of `Student` class and pass it as the argument to the constructor of superclass like `super(age, name, parent_permission)`.

Reply:
Hi @SajjadAhmad14,

Thank you for the review.

However, adding `parent_permission = true` as a parameter of both the `Student` and `Teacher` class would throw an error when you're creating an new instance of the classes.

Attached is a screenshot that was taken from my editor after adding `parent_permission = true` and creating a new instance in `irb`.

![](https://i.imgur.com/0VjgxxY.png)

Kindly check.

Thanks. 👍